### ISSUE TITLE
Accessibility Discussion: Focus Exact Item In Status Bar

### ISSUE THREAD

**Issue by jhomme** [microsoft/vscode#101205]
Hi,
Is it possible to count on the positions of items in the status line? Failing that, is it possible to query information that VS Code knows about even if it's not in a fixed position on the status line? I'm asking this because I noticed that in my setup, line and column number information is the next to last item. I have VS Code Extensions that show other information, such as word count and reading level. I would like to set up key strokes that get the individual pieces of information I'm interested in and avoid reading through all the information on the status line to find what I want to know. I'd like to be able to count on always finding line and column number, for instance, or how many spelling errors are in my Markdown file. Now that I think of it, it makes sense to be able to find these things out independent of the status line. I'd like to do it quickly through key strokes, because a quick key stroke allows me to keep the flow of my work going better than stopping, finding something on the status line after F6ing and then finding my way back and reorienting myself to what I was doing.

**Comment by isidorn** [comment]
@jhomme hi! Thanks for creating this issue. I think it is a fair feature request against the status bar to make accessing an exact item possible.
However we currently do not have this on the plan, as for most items there are alternative command palette commands on how to get that information.
For example for line numbers that is `Ctrl + G`. For problems it is `Ctrl + Shit + M` and so on.

For now leaving this open as a feature request and we might tackle in the future.
